src/themes/elife/styles.css

```diff
@@ -92,3 +94,12 @@ main {
     }
   }
 }
+
+:--datePublished {
+  font-family: 'Noto Sans', Arial, Helvetica, sans-serif;
+  font-size: calc(var(--text-size-small) * 1rem);
+  line-height: calc(1.5rem / var(--text-size-small));
+  letter-spacing: 0.5px;
```

Comment:
Not sure whether this should be a custom property 🤔

Reply:
It could be as all the letter-spacing variables in pattern library are the same. So will add.